src/Surfnet/StepupMiddlewareClient/Identity/Service/RaCandidateService.php

```diff
@@ -50,8 +50,8 @@ public function search(RaCandidateSearchQuery $query)
      * @param string $identityId
      * @return array|null
      */
-    public function get($identityId, $institution)
+    public function get($identityId, $institution, $actorId)
     {
-        return $this->apiClient->read('ra-candidate/%s/%s', [$identityId, $institution]);
+        return $this->apiClient->read('ra-candidate/%s/%s?actorId=%s', [$identityId, $institution, $actorId]);
```

Comment:
As I mentioned in my MW review, this API change needs to be updated in the docs & Postman config.